docs/src/man/subsets.md

```diff
@@ -6,7 +6,7 @@ The `DataArray` type is meant to behave like a standard Julia `Array` and tries
 
 One dimensional `DataArray`:
 
-```julia
```

Comment:
OK. I think @MichaelHatherly said he wanted these to work as doctests, with automatic checking of the results, but it might not work at the moment.

Does code highlighting work though?